src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -113,6 +114,7 @@
     + "    col_index INTEGER NOT NULL,\n"
     + "    col_name TEXT,\n"
     + "    col_type TEXT NOT NULL,\n"
+    + "    is_indexed INTEGER NOT NULL, \n" 
```

Comment:
The easiest thing I could think of would be to let users specify the CreateIndex encoding as the following:

{
  "relationKey": {
    "userName": "jwang",
    "programName": "global_join",
    "relationName": "smallTable"
  },
  "schema": {
    "columnTypes": [
      "LONG_TYPE",
      "LONG_TYPE"
    ],
    "columnNames": [
      "col1",
      "col2"
    ]
  },
  "indexes": [
    0,1
  ],
  "indexIsAscending": [0,1]
}

Perhaps "indexIsAscending" isn't the best name, but ultimately it would create an index like the following:

"CREATE INDEX indexName ON "jwang:global_join:smallTable" (col1 DESC, col2 ASC)"